### PR TITLE
A new script to get rank wise google search links added

### LINKED ISSUE #608: A script to get a list of links related to a particular query (Google Search Rank wise)
Is your feature request related to a problem? Please describe.
You can use this script to get the links related to a particular search query (rank wise), which can be used by the user to get to any link without scrolling down or using next to move to next page.

Describe the solution you'd like
Using a google api, beautifulsoup4 and urllib libraries in Python to store all the links in a list and displaying the list as output.

Describe alternatives you've considered

Additional context

### PR DESCRIPTION
**Pull request by shiveshchoudhary**
# Description

A new directory named "Query_Script" added in the Python directory, this directory contains three files a python script to get rank wise link results, a requirement.txt file, and a readme.MD

Fixes #608 

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`

### CODE REVIEW

File: Python/Query_Script/query_script.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+			break
+
+		print (count,link,"\n")
+		count+=1
+		
+
+if __name__ == "__main__":
+	main()
+
+
```
Comment (HarshCasper) [inline Python/Query_Script/query_script.py:24]: Remove extra lines of Code.

File: Python/Query_Script/Readme.MD [hunk 2]
```diff
@@ -0,0 +1,16 @@
+### Python Script to get the links of a search query Google Search rank wise
+
+### Modules used
+
+#### Google
```
Comment (HarshCasper) [inline Python/Query_Script/Readme.MD:5]: We don't need so many Headings. Follow an elegant structure by taking a look at other Python Scripts.

File: Python/Query_Script/query_script.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+	for links in search(keyword, tld= domain, pause= 2):
+		yield links
+		
+def main():
+	query = input("Enter your search keyword: ")
+	number_of_results = int(input("Enter number of links you want: "))
+	count =1
+	for link in google_search_results(query):
+		# final result is printed in the form [rank link]
+		if count> number_of_results:
```
Comment (HarshCasper) [inline Python/Query_Script/query_script.py:14]: Use proper formatting. Run your Code through `PyLint` to find issues in your Code and fix them up.

File: Python/Query_Script/query_script.py [hunk 4]
```diff
@@ -0,0 +1,24 @@
+from googlesearch import search
+
+def google_search_results(keyword, domain= 'com'):
+	#generator function
```
Comment (HarshCasper) [inline Python/Query_Script/query_script.py:4]: Use proper Docstrings.

**Review by HarshCasper — CHANGES_REQUESTED** [review]
Are the dependencies really that huge? 

I see that you are using just `googlesearch` for the purpose. You can use `pip show` to better take a look at the dependencies.

File: Python/Query_Script/query_script.py (hunk 1 above)
Comment (shiveshchoudhary) [reply Python/Query_Script/query_script.py]: Hi I have updated the code, pls check

File: Python/Query_Script/query_script.py (hunk 3 above)
Comment (shiveshchoudhary) [reply Python/Query_Script/query_script.py]: check

File: Python/Query_Script/Readme.MD (hunk 2 above)
Comment (shiveshchoudhary) [reply Python/Query_Script/Readme.MD]: I have followed the readme style of top scripts in python directory, pls tell if any more changes are required

File: Python/Query_Script/query_script.py (hunk 3 above)
Comment (shiveshchoudhary) [reply Python/Query_Script/query_script.py]: > Are the dependencies really that huge?
> 
> I see that you are using just `googlesearch` for the purpose. You can use `pip show` to better take a look at the dependencies.

Sorry for that issue I have updated it, now it has only 4 dependencies

**Comment by HarshCasper** [comment]
LGTM! Thanks 👍

**Pull request merged**